verification/curator-service/api/src/controllers/auth.ts

```diff
@@ -212,14 +212,14 @@ export class AuthController {
          */
         this.router.post(
             '/request-password-reset',
-            async (req: Request, res: Response): Promise<void> => {
+            async (req: Request, res: Response): Promise<Response<any>> => {
                 const email = req.body.email as string;
 
                 try {
                     // Check if user with this email address exists
                     const user = await User.findOne({ email });
```

Comment:
The behaviour here means that if a user signed up with Gmail, they can still get a password reset link. I think that means that they'll get "converted" to a password user, when they use the link to reset their password. Is that expected? Will Google OAuth2 still work?